### PR TITLE
Readme updates + yaml update

### PR DESCRIPTION
**Pull request by TonyRice**

### CODE REVIEW

**Comment by TonyRice** [comment]
This currently lies on https://github.com/storyscript/platform-engine/pull/401

**Comment by TonyRice** [comment]
This needs some more updates.

**Comment by TonyRice** [comment]
@judepereira can you check this out? Needed for latest file service changes.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+# Read the file just written to as a raw data
+str = file read path:"/tmp/foo" binary: true
+
+# Read the file just written to as a string
+str = file read path:"/tmp/foo" binary: false
+
+# Read and write raw bytes
+http server as client
+  when client listen method:"post" path:"/" as r
+    # the file service receives the zipe file,
```
Comment (wilzbach) [inline README.md]: zip

Comment (TonyRice) [reply README.md]: Good catch

**Pull request merged**